spec/controllers/api/public/v1/providers/courses_controller_spec.rb

```diff
@@ -60,5 +60,37 @@
         expect(JSON.parse(response.body)["data"].size).to eql(2)
       end
     end
+
+    describe "include" do
+      let(:course) { create(:course, :with_accrediting_provider, provider: provider) }
+      before do
+        course
+      end
+
+      it "returns the provider and accrediting body connected to the course" do
+        get :index, params: {
+          recruitment_cycle_year: provider.recruitment_cycle.year,
+          provider_code: provider.provider_code,
+          include: "provider,accredited_body",
+        }
+
+        expect(JSON.parse(response.body)["data"][0]["relationships"].keys).to include("provider")
+        expect(JSON.parse(response.body)["data"][0]["relationships"].keys).to include("accredited_body")
+        expect(JSON.parse(response.body)["included"][0]["id"]).to eql(course.accrediting_provider.id.to_s)
+        expect(JSON.parse(response.body)["included"][0]["type"]).to eql("providers")
+        expect(JSON.parse(response.body)["included"][1]["id"]).to eql(provider.id.to_s)
+        expect(JSON.parse(response.body)["included"][1]["type"]).to eql("providers")
+      end
+
+      it "doesn't include subjects as they aren't serialized" do
+        get :index, params: {
+          recruitment_cycle_year: provider.recruitment_cycle.year,
+          provider_code: provider.provider_code,
+          include: "subjects",
+        }
+
+        expect(JSON.parse(response.body)["data"][0]["relationships"].keys).to_not include("subjects")
```

Comment:
@darokel maybe we should do some slice and dice so this PR can benefit

https://github.com/DFE-Digital/teacher-training-api/blob/5748d7c3b8e9de80e701bf3f869899d8cd818833/spec/support/request_helper.rb#L1-L5

Reply:
yeah good shout